docs/experimentConfigReadme.md

```diff
@@ -126,6 +127,7 @@ targets = [
         "id": "simple_target",
         "destSpace" : "player",                 // This is a player-centered-spherical-space target
         "visualSize" : [0.5, 0.5],              // 0.5m size
+        "colors": [Color3(0,1,0), Color3(1,0,0)];   // Green at max health, red at min health
```

Comment:
Usually we put the defaults here. If these are the defaults, then that means that the target's set colors will always override what you set in the experiment or session unless you manually set this to a 0 size array.

Reply:
Seeing this in context, it's just one target sample among a group of samples, most of which don't set the `colors` parameter. I believe this is fine to keep as is.